### PR TITLE
updated forms

### PR DESCRIPTION
**Pull request by BuckG71**
* added autofocus to text fields
* made minor formatting adjustments to html templates

### CODE REVIEW

File: bridge/bridgeapp/forms.py [hunk 1]
```diff
@@ -10,24 +10,27 @@ class ResponseForm(forms.Form):
         required=True,
         label='',
         widget=forms.Textarea(attrs={
-            'rows': '2',
+            'rows': '3',
```
Comment (Sanlung) [inline bridge/bridgeapp/forms.py:13]: I believe the number of rows or just how the input field will appear is a determination belonging to the styling stage of the app. Anything done now is subject to change once the style are discussed, so why bother now?

File: bridge/bridgeapp/forms.py [hunk 2]
```diff
@@ -10,24 +10,27 @@ class ResponseForm(forms.Form):
...
+        }))
 
 
 class ThreadForm(forms.Form):
     body = forms.CharField(
         required=True,
         label='',
         widget=forms.Textarea(attrs={
-            'rows': '2',
+            'rows': '3',
```
Comment (Sanlung) [inline bridge/bridgeapp/forms.py:25]: Please see comment on line 13 in the above.

File: bridge/bridgeapp/forms.py [hunk 3]
```diff
@@ -10,24 +10,27 @@ class ResponseForm(forms.Form):
...
             'cols': '50',
-            'placeholder': 'Post a question here.'
-            }))
+            'autofocus': True,
```
Comment (Sanlung) [inline bridge/bridgeapp/forms.py:27]: Can you give example of a site that auto-focus input fields other than on the login/sign up page?

File: bridge/bridgeapp/templates/category.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {% extends 'home.html' %} {% block main %}
-<h2>{{ category.type }}</h2>
+<h2><u>{{ category.type }}</u></h2>
```
Comment (Sanlung) [inline bridge/bridgeapp/templates/category.html:2]: As to the <u> tag, all HTML/CSS literature would suggest text decoration or emphases be done via CSS, not HTML. Besides, this is a style done prematurely and subject to change in my opinion.

File: bridge/bridgeapp/templates/home.html [hunk 5]
```diff
@@ -11,6 +11,7 @@
         <header>
             <nav>
                 <a href="{% url 'home' %} "><img src="/static/images/bridge.png" /></a>
+                <h2>Spanning Questions And Answers</h2>
```
Comment (Sanlung) [inline bridge/bridgeapp/templates/home.html:14]: I don't understand what you're doing here. I see a <nav> with links lined up horizontally, as I explained here: https://justicethroughcode.slack.com/archives/C03B8N27JG7/p1653113798388519.

File: bridge/bridgeapp/templates/thread.html [hunk 6]
```diff
@@ -1,9 +1,9 @@
 {% extends 'home.html' %} {% block main %}
-    <h1>Question and Responses:</h1>
+    <h2>/ Selected Thread /</h2>
```
Comment (Sanlung) [inline bridge/bridgeapp/templates/thread.html:2]: Premature.

File: bridge/bridgeapp/templates/thread.html [hunk 7]
```diff
@@ -1,9 +1,9 @@
...
     <h3>{{ thread.body }}</h3>
     <ul>
         {% for response in responses %}
-            <li>{{ response.body }}&nbsp;<a href="{% url 'thread' thread_id=thread.id resp_id=response.id %}">Edit</a></li>
+            <li>{{ response.body }} &nbsp;<a href="{% url 'thread' thread_id=thread.id resp_id=response.id %}">(...edit response)</a></li>
```
Comment (Sanlung) [inline bridge/bridgeapp/templates/thread.html:6]: Premature.

**Review by Sanlung — COMMENTED** [review]
I believe a majority of the changes are premature and will be eliminated once the app gets styled.

Myself would prioritize before MVP grading, in the order:
1. complete a README that pass the rubrics;
2. complete commenting that conform to standard;
3. finding a Django native way to prompt the user to pick at least one category when none checked, and, if not, find a non-native way to do it after MVP grading; and
4. giving a rudimentary styling to make the app appear in a rough shape of header-content-footer across the pages.

But I'm tied up with potential employers/jobs and current obligations, so I'll go along with what you got to offer.

**Review by rlee794 — APPROVED** [review]

**Review by ponyNYC — APPROVED** [review]

**Pull request merged**